modules/multiqc/multiqc.nf

```diff
@@ -1,13 +1,13 @@
 nextflow.enable.dsl = 2
 
-params.fastqcResultsDir = "${params.outdir}/fastqc"
-params.resultsDir = "${params.outdir}/multiqc"
-params.saveMode = 'copy'
-params.shouldPublish = true
+params.fastqc_resultsDir = "${params.outdir}/fastqc"
```

Comment:
Missed this one.

Reply:
true! i forgot to check the params file and the config, just a second

Reply:
@abhi18av the config and the params files seems ok to me, the dandrea pipeline. but I need to know the file names for the kraken params, to update the mock data generator and also the stub params

Reply:
Good question 👍 

1. `mkdir ./test_data/mock_data/mock_kraken_db_dir` <- command in mock file generator
2. `--kraken2_db test_data/mock_data/mock_kraken_db_dir` <- parameter for the stub in Makefile


Also, I think that the `workflows` folder might also need to the same changes since the `classify_taxonomy` is there among others.

Reply:
@abhi18av I don't understand, what changes the workflow needs? :pensive:

Reply:
The changes regarding the snake_case from camelCase

Reply:
I've already did it please check the workflow [here](https://github.com/bioinformatics-lab/dandrea_legionella_nf/blob/d11abcacbf5dc437658b8eea12895a7921eb110b/workflows/classify_taxonomy/classify_taxonomy.nf), I'll update the other files now

Reply:
@abhi18av I added this parameter to the stub profile, as we were doing before, and also updated the mock file generator so this part is good to go and I'll resolve this conversation

Reply:
@Mxrcon thanks for taking the initiative.

TIP: Let the maintainer resolve (or keep) the issue.

Reply:
ok! I'll let the hard work to you! :smirk: